### ISSUE TITLE
VQ VAE model's reconstruction is a black image

### ISSUE THREAD

**Issue by ghost** [AntixK/PyTorch-VAE#51]
I am using your wonderful library in my research project. There seems to be a bug in the VQ VAE mode class where the reconstruction is a blank image. Is it a known bug? Can you please help me with this issue?

**Comment by manupillai308** [comment]
I m also facing the same issue.

**Comment by jiang-wenbin** [comment]
You should monitor the loss. If the loss grow up dramatically, reduce the learning rate. I set the LR 0.0003

**Comment by ghost** [comment]
I will try @jiang-wenbin approach. Marking this closed.
